core/src/main/resources/lib/form/enum.jelly

```diff
@@ -36,9 +36,14 @@ THE SOFTWARE.
       The name of the enum to set as default value for the first configuration.
     </st:attribute>
   </st:documentation>
-  <select class="setting-input" name="${field}">
-    <j:forEach var="it" items="${descriptor.getPropertyType(instance,field).enumConstants}">
-      <f:option value="${it.name()}" selected="${instance != null ? it==instance[field] : it.name()==default}">
+
+  <j:if test="${attrs.field==null}">
+    <j:set target="${attrs}" property="field" value="${entry.field}" />
```

Comment:
It would be better to introduce a new local variable. Overriding attribute fields might be working now, but I was unable to find Jelly documentation about whether it is a supported behavior. It may stop working eventually.

Reply:
This is the same code what is used in `core/src/main/resources/lib/form/descriptorList.jelly` or `prepareDatabinding.jelly`. The descriptorList is there for years so I treated it as a trusted example.
Also Jelly developments seems dead (no single commit for almost 3 years) so I don't think there is a risk of stopping to work.
Of course If you think that using local variable is more wise then please modify the code how you fell is the best. You are the top contributor so I trust your judgement :)